server.js

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
```

Comment:
More accurately, creating CLI is fine, but I don't want it to be included in this repository. I really agree with https://github.com/sindresorhus/ama/issues/17#issuecomment-117860283.

Feel free to create a new npm package `pkg-bin-server` and use it.